### PR TITLE
Fixes for python3 compatibility

### PR DESCRIPTION
**Pull request by kiddinn**
This round mainly focuses on use of decode. 

Part of #823

### CODE REVIEW

**Comment by kiddinn** [comment]
Not requesting code review right now, still doing tests, also wanted to have travis run on the code first.

File: timesketch/lib/py2to3.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""The Python 2 and 3 compatible type definitions.
+
+File copied from the plaso project.
+"""
+import sys
```
Comment (Onager) [inline timesketch/lib/py2to3.py]: Don't use this, use six instead, it has all the same functionality.

Comment (kiddinn) [reply timesketch/lib/py2to3.py]: ack.... much better to just use six... done that

**Comment by kiddinn** [comment]
removed py2to3 in favor of just using six... ready for a review

**Pull request merged**